Source/UnrealGT/Private/GTRandomColorGenerator.cpp

```diff
@@ -0,0 +1,45 @@
+#include "GTRandomColorGenerator.h"
+
+#include "Math/Color.h"
+
+TArray<int> MakeSequentialIndexArray(const int FirstIndex, const int LastIndex)
+{
+    TArray<int> arr;
+    arr.Reserve(255);
+    for (auto i = FirstIndex; i <= LastIndex; i++)
+        arr.Emplace(i);
+    return arr;
+}
+
+void ShuffleArray(TArray<int>& Arr)
+{
+    for (int i = 0; i < Arr.Num(); ++i)
+    {
+        int Index = FMath::RandRange(i, Arr.Num() - 1);
+        if (i != Index)
+        {
+            Arr.Swap(i, Index);
+        }
+    }
+}
+
+TArray<int> MakeShuffledIntArray(const int LowestIndex, const int HighestIndex)
+{
+    TArray<int> IDs = MakeSequentialIndexArray(LowestIndex, HighestIndex);
+    ShuffleArray(IDs);
+    return IDs;
+}
+
```

Comment:
I prefer to wrap helpers in an anonymous namespace `namespace {}` to avoid linker issues.

Reply:
I do too. Done.